R/pkg/R/utils.R

```diff
@@ -545,10 +545,6 @@ processClosure <- function(node, oldEnv, defVars, checkedFuncs, newEnv) {
               found <- sapply(funcList, function(func) {
                 ifelse(identical(func, obj), TRUE, FALSE)
               })
-              if (sum(found) > 0) {
```

Comment:
Discussed offline. The change would lead to dead loop. We should make sure `newEnv` contains the (cleaned) node.

Reply:
Is the dead loop case when the same function with the same environments is recursively called in the closure?